packages/node-core/src/indexer/fetch.service.ts

```diff
@@ -379,8 +398,11 @@ export class FetchService<DS extends BaseDataSource, B extends IBlockDispatcher<
     return this.latestFinalizedHeight;
   }
 
-  resetForIncorrectBestBlock(blockHeight: number): void {
```

Comment:
It would be worth checking that other SDKs dont use this function. But it probably can be removed.